CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ if(opencoarrays_aware_compiler)
   add_mpi_test(get_with_offset_1d 2 ${tests_root}/unit/send-get/get_with_offset_1d)
   add_mpi_test(whole_get_array 2 ${tests_root}/unit/send-get/whole_get_array)
   add_mpi_test(strided_get 2 ${tests_root}/unit/send-get/strided_get)
+  add_mpi_test(strided_sendget 4 ${tests_root}/unit/send-get/strided_sendget)
```

Comment:
I agree: oversubscribing is a great way to identify problems, including the [bug report](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=78505) that I just submitted on gfortran last night.